.github/pull_request_template.md

```diff
@@ -0,0 +1,45 @@
+## Description
+<!-- Add a description here -->
+
+### Documentation sets edited in this PR
+
+_Check all that apply._
+
+- [ ] Stateful (`docs/en/observability/*`)
+- [ ] Serverless (`docs/en/serverless/*`)
+- [ ] Integrations Developer Guide (`docs/en/integrations/*`)
+- [ ] None of the above
+
+### Related issue
+Closes # <!-- Add the issue this PR closes here -->
+
+## Checklist
+
+<!--
+Add labels to:
+1. Backport to other versions (`backport-*`):
+    - `backport-8.x` to backport to the latest minor
```

Comment:
Last question... do you think we need to be more specific with this? 

```suggestion
    - `backport-8.x` to backport to the latest minor branch (this may be a yet-to-be-released version)
```

Reply:
>latest minor branch

This might be the wrong phrase because (for example) we don't currently have an 8.16 branch.

Reply:
I don't think so. We're not doing anything novel in obs-docs so I'm inclined to leave it as-is.